### ISSUE TITLE
Pytest emits a warning

### ISSUE THREAD

**Issue by bochecha** [dbader/pytest-mypy#25]
I'm getting the following warning from pytest when using this project:

```
pipenv run py.test --mypy -m mypy
======================================================== test session starts ========================================================
platform linux -- Python 3.7.3, pytest-4.5.0, py-1.8.0, pluggy-0.11.0
rootdir: /home/mathieu/Projects/endless/chuleisi, inifile: setup.cfg
plugins: mypy-0.3.2, flakes-4.0.0, cov-2.7.1, codestyle-1.4.0
collected 11 items / 4 deselected / 7 selected                                                                                      

[… snip …]

========================================================= warnings summary ==========================================================
venv/lib/python3.7/site-packages/_pytest/mark/structures.py:324
  venv/lib/python3.7/site-packages/_pytest/mark/structures.py:324: PytestUnknownMarkWarning: Unknown pytest.mark.mypy - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

I guess this is related to what was done in #18?

**Comment by thatneat** [comment]
Looks like you're currently making a ton of progress on a more significant release. If that's going to be a few days out, is there any chance we can get a bugfix release for this on PyPI in the meantime?

**Comment by dmtucker** [comment]
Yes, I'll try to get that out today.
